packages/modules/pgr/src/components/Modal.js

```diff
@@ -35,9 +38,13 @@ const Modal = (props) => {
   //   }, [file]);
 
   useEffect(async () => {
+    setError(null);
     if (file) {
-      const response = await Digit.UploadServices.Filestorage(file);
-      setUploadedFile(response.data.files[0].fileStoreId);
+      if (file.size > 5242880) {
+        setError("Maximum upload size exceeded");
+      }
+      const response = await Digit.UploadServices.Filestorage(file, cityDetails.code);
+      setUploadedFile(response?.data?.files[0]?.fileStoreId);
```

Comment:
you need to show error if response or data or files is null, that you have checked here

Reply:
I did eariler but then I remove it bcz If I add else here, then at later period of time we will never know that there is an error happening on network tab bcz that code never run bcz of above limit check.